setup.py

```diff
@@ -67,13 +67,12 @@ def is_requirement(line):
     sys.exit()
 
 README = open(os.path.join(os.path.dirname(__file__), 'README.rst')).read()
-CHANGELOG = open(os.path.join(os.path.dirname(__file__), 'CHANGELOG.rst')).read()
 
 setup(
```

Comment:
I don't think we need this file at all?  Will there be any Python code in this repo?